tools/tpm2_tool.c

```diff
@@ -139,7 +139,11 @@ int main(int argc, char *argv[], char *envp[]) {
     /* TODO SAPI INIT */
     TSS2_SYS_CONTEXT *sapi_context = sapi_ctx_init(tcti);
 
-    tpm2_errata_init(sapi_context);
+    if (!flags.disable_errata) {
+        tpm2_errata_init(sapi_context);
+    } else {
+       LOG_INFO("Errata disabled");
```

Comment:
And this could also be removed if we flip the logic.

Reply:
Agreed.